agent/xds/server.go

```diff
@@ -152,7 +152,7 @@ type Server struct {
 
 	DisableV2Protocol bool
 
-	activeStreams activeStreamCounters
+	activeStreams *activeStreamCounters
 }
 
 // activeStreamCounters simply encapsulates two counters accessed atomically to
```

Comment:
Is this comment still accurate? Maybe we could add a small explanation why `activeStreams` is a pointer

Reply:
Yeah I guess I should adjust that comment slightly.